src/main/kotlin/com/cognifide/gradle/aem/environment/docker/Container.kt

```diff
@@ -89,6 +88,9 @@ class Container(val docker: Docker, val name: String) {
             }
         }
 
+    val up: Boolean
+        get() = running && isLocked(LOCK_UP)
```

Comment:
`up` is just more strong `running` which means that containers are running and they are configured by GAP DSL